x-pack/platform/plugins/shared/onechat/public/application/hooks/use_conversation.ts

```diff
@@ -32,7 +32,7 @@ const useConversation = () => {
     },
   });
 
-  return { conversation };
+  return useMemo(() => ({ conversation }), [conversation]);
```

Comment:
Do we really need this? From my understanding stuff returned from `useQuery` are supposed to be stable?